### PR TITLE
Website edits round 2

### PR DESCRIPTION
**Pull request by khangmach**
Big graphical and layout changes to LE home and Kolibri page

### CODE REVIEW

**Comment by khangmach** [comment]
@jtamiace @jamalex up for review and merge

**Comment by khangmach** [comment]
Needs to also be merged with #333

File: fle_site/apps/main/static/css/_kickstarter_mobile.css [hunk 1]
```diff
@@ -27,14 +27,16 @@
...
 	}
 /*	.below-fold2 {
 		min-height: 528px;
 		height: 50vh;
 	}*/
 	.fle_title_bold,
 	.fle_title h2 {
-		font-size: 35px;
+		font-size: 28px;
+		/*max-width:700px;*/
```
Comment (jtamiace) [inline fle_site/apps/main/static/css/_kickstarter_mobile.css]: Remove dead code

File: fle_site/apps/main/templates/main/translate.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+            <h4>Time Commitment</h4>
+            <p>Our translators put in an average of 30 minutes to one hour of work per day for a turnaround time of two weeks for a complete translation.<br><br>We ask that a language package is finished within a maximum of three weeks. For the four weeks following translation completion, we ask that translators be available in case any additional updates need to be made. How the work fits in your schedule is completely up to the translator.
+            <br><br>
+            <strong>Official guidelines are coming soon!</strong> <br><br>For more information, contact us at <a href="mailto:i18n@learningequality.org">i18n@learningequality.org</a> and we will be happy to help. 
+            </p>
+
+        </div>
+
+
+        <!-- Contact Section -->
```
Comment (jtamiace) [inline fle_site/apps/main/templates/main/translate.html]: Should remove if we don't plan on including this again

File: fle_site/apps/main/templates/main/partials/_kickstarter_intro2.html [hunk 3]
```diff
@@ -16,23 +16,38 @@
...
+                        <img class="use-image" src="/static/img/kickstarter/use_cases/nonformal.png">
+                        <p>Non-formal school programs </p>
+                    </div>
+                    <div class="col-md-4 col-xs-6 col-sm-6 use shift-down">
+                        <img class="use-image" src="/static/img/kickstarter/use_cases/prisons.png">
+                        <p>Prisons</p>
+                    </div>
+                    
                 </div>
+                <!--<div class="row use-cases ">
```
Comment (jtamiace) [inline fle_site/apps/main/templates/main/partials/_kickstarter_intro2.html]: Should remove if we don't want to include it again

File: fle_site/apps/main/templates/main/kickstarter.html [hunk 4]
```diff
@@ -105,48 +99,86 @@
...
+                </div>
+            </div>
+        </div>
 
 
         <!-- Contact Section -->
-        <section id="contact" class="not-parallax-section">
-            {% include "main/partials/_kickstarter_footer.html" %}
-        </section>
+        <!--<section id="contact" class="not-parallax-section">
```
Comment (jtamiace) [inline fle_site/apps/main/templates/main/kickstarter.html]: Should remove if we don't plan on including it again

File: fle_site/apps/main/templates/main/partials/_kickstarter_intro2.html (hunk 3 above)
Comment (jtamiace) [inline fle_site/apps/main/templates/main/partials/_kickstarter_intro2.html]: Should remove if we don't plan on including again

File: fle_site/apps/main/templates/main/translate.html (hunk 2 above)
Comment (jtamiace) [inline fle_site/apps/main/templates/main/translate.html]: Should remove if we don't plan on including again

File: fle_site/apps/main/templates/main/homepage.html [hunk 5]
```diff
@@ -34,64 +34,55 @@
 
 <!-- ============= MISSION SCREEN ================ -->
 		<div class="row mission-screen">
-			<div id="learn-more" class="col-xs-10 col-sm-8 col-md-8 col-lg-8 mission-box">
+			<div id="learn-more" class="col-xs-10 col-sm-10 col-md-10 col-lg-10 mission-box">
+				<!--<img src="{% static 'img/world-map-adjusted.png'%}" class="children-ratio center-block" alt="children">-->
```
Comment (jtamiace) [inline fle_site/apps/main/templates/main/homepage.html]: Remove if code is dead

File: fle_site/apps/main/templates/main/partials/_kickstarter_features_tiledesign.html [hunk 6]
```diff
@@ -4,61 +4,49 @@
...
-		</div>
-		<span id="kolibri-name">Kolibri</span>
-		<div>
-			<span class="slogan">Learn.</span>
-			<span class="slogan">Create.</span>
-			<span class="slogan">Share.</span>
-		</div>
-	</div>
-	<div class="col-md-7" id="tile-container">
+<!--<div class="container-fluid" id="feature-container">
```
Comment (jtamiace) [inline fle_site/apps/main/templates/main/partials/_kickstarter_features_tiledesign.html]: Dead code?

**Comment by jtamiace** [comment]
Reason for quotations? Feel like it breaks the flow of reading a little bit since we're not quoting anybody, and the "speaker" feels different across these two things for example

screen shot 2017-10-16 at 10 49 21 am

screen shot 2017-10-16 at 10 59 35 am

**Comment by jtamiace** [comment]
One of the pieces of feedback I got a few years ago is that the graphics (white skin complexion, straight hair) does not apply to everyone around the world... just wondering if they should be altered to reflect that

screen shot 2017-10-16 at 11 01 19 am

**Comment by jtamiace** [comment]
Wondering if this section should be "squeezed" to match the width of the other sections?

screen shot 2017-10-16 at 11 04 46 am

**Comment by jtamiace** [comment]
Really good visual enhancement overall!

**Comment by jtamiace** [comment]
(Doesn't have to be now at all, just want to put this out there)
If imps team has any case studies of how this works, could help the general population understand how it works

screen shot 2017-10-16 at 11 45 56 am

**Comment by jtamiace** [comment]
Changes look good to me to merge. @jamalex ?

File: fle_site/apps/main/templates/main/partials/_kickstarter_features_tiledesign.html [hunk 7]
```diff
@@ -115,11 +57,58 @@
...
+	                            	<li>Life skills materials on subjects such as health, safety, trauma recovery, digital literacy, and others</li>
+	                            </ul>
+
+	                            <p>Some of our content sources include:</p>
+	                            <a href="https://www.khanacademy.org/" target="_blank"><img src="{% static 'img/kickstarter/khan-academy.png' %}"></a>
+	                            <a href="https://www.aflatoun.org/" target="_blank"><img src="{% static 'img/kickstarter/Aflatoun.png' %}"></a>
+	                            <a href="http://www.africanstorybook.org/" target="_blank"><img src="{% static 'img/kickstarter/African-storybook.png' %}"></a>
+	                            <a href="https://www.librarieswithoutborders.org/" target="_blank"><img src="{% static 'img/kickstarter/Libraries-without-borders.png' %}"></a>
+	                            <a href="http://prathambooks.org/" target="_blank"><img src="{% static 'img/kickstarter/Pratham-books.png' %}"></a>
+	                            <a href="https://www.sikana.tv/ru" target="_blank"><img src="{% static 'img/kickstarter/Sikana.png' %}"></a>
```
Comment (jamalex) [inline fle_site/apps/main/templates/main/partials/_kickstarter_features_tiledesign.html]: Any particular reason we link to the Russian version?

Comment (jamalex) [reply fle_site/apps/main/templates/main/partials/_kickstarter_features_tiledesign.html]: I'll update directly here.

**Pull request merged**